app/controllers/api/v1/party/players_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Api::V1::Party::PlayersController < ApplicationController
+
+  before_action :logged_in?
+
+  def create
+    # binding.pry
+    # Request Validations
+    if !params.keys.include?("player_id")
+      render json: { "error": "a valid player_id parameter is required for this request"}, status: 400
+    elsif (params["player_id"] =~ /\d+/) == nil
```

Comment:
Fancy syntax here! I'm not to familiar with this `\d` thing, care to explain how it works/what's being checked here?